src/SharedFrameworkValidation/SharedFrameworkValidation.proj

```diff
@@ -57,16 +56,15 @@
 
   <Target Name="BuildPackagesProject">
     <PropertyGroup>
-      <_RestorePackagesCommand>"$(DotnetToolCommand)" restore $(MSBuildThisFileDirectory)/RestoreSDKProject/RestoreSDKProject.csproj /p:_PublishRID=$(_PublishRID) /p:_RuntimeFrameworkVersion=$(MicrosoftNETCoreAppPackageVersion) /p:CoreFxExpectedPrerelease=$(CoreFxExpectedPrerelease)</_RestorePackagesCommand>
```

Comment:
We still need to pass this in since the project we are building doesn't import dir.props and doesn't know about the regular properties like `TargetRID`